packages/electron-updater/src/DebUpdater.ts

```diff
@@ -31,7 +31,9 @@ export class DebUpdater extends BaseUpdater {
     const sudo = this.wrapSudo()
     // pkexec doesn't want the command to be wrapped in " quotes
     const wrapper = /pkexec/i.test(sudo) ? "" : `"`
-    const cmd = ["dpkg", "-i", options.installerPath, "||", "apt-get", "install", "-f", "-y"]
+    // application artifact names may includes spaces in their name which will cause error when executing install command.
```

Comment:
Please consider changing it to this for grammatical correct-ness.
```suggestion
    // application artifact names may include spaces in their name which leads to an error when executing install command
```

Reply:
Done.